### PR TITLE
Added Jetshop support

### PR DESCRIPTION
**Pull request by robertkvant**
Added support for the Jetshop e-commerce system (http://jetshop.se)

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -9690,6 +9690,14 @@
 			},
 			"website": "http://notenbomer.nl/Producten/Content_management/io4_|_cms"
 		},
+		"Jetshop": {
+			"cats": [
+				"6"
+			],
+			"html": "(?:<div id=\"jetshop-branding\">|<script>var JetshopData={|jetshop(?:Api|Image|Checkout)BaseUrl:|sr-jetshop-wp)",
```
Comment (jvoisin) [inline src/apps.json]: Please split this regex in an array.

Comment (jvoisin) [inline src/apps.json]: Please put `JetshopData` into an `env` object.

Comment (jvoisin) [inline src/apps.json]: Don't you think that `sr-jetshop-wp` is too false-positive prone? Can't you refine it a bit?

Comment (robertkvant) [inline src/apps.json]: Sounds like sensible suggestions

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**